examples/noisy_simulation_example.py

```diff
@@ -0,0 +1,36 @@
+"""
+Creates and simulate a noisy circuit using cirq.ConstantQubitNoiseModel class.
+"""
+import cirq
+
+
+def noisy_circuit_demo(amplitude_damp):
+    """Demonstrates a noisy circuit simulation.
+    """
+    q = cirq.NamedQubit('q')
+    noisy_circuit = cirq.Circuit.from_ops(
```

Comment:
Just `circuit`. The circuit itself is a noiseless spec.

Reply:
Done.